PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskPID.cxx

```diff
@@ -1270,29 +1270,29 @@ void AliAnalysisTaskPID::UserCreateOutputObjects()
   }
   
   if (GetDoTPCclusterStudies()) {
-    Int_t nBinsPrime = 320;
-    Double_t primeLow = 0.4;
-    Double_t primeHigh = 2.0;
+    const Int_t nBinsPrime = 320;
+    const Double_t primeLow = 0.4;
+    const Double_t primeHigh = 2.0;
     const Int_t arrNumbers = 4;
     fTPCclusterStudies = new TObjArray*[arrNumbers];
     const Int_t histNumbers = 3;
     
-    const Int_t nBinsY[histNumbers+1] = {8, 4, 11, 15};
-    Float_t twopi = 2.0 * TMath::Pi();
+    constexpr Int_t nBinsY[histNumbers+1] = {8, 4, 11, 15};
+    constexpr Float_t twopi = 2.0 * TMath::Pi();
```

Comment:
@MartinSchmidtTuebingen @pzhristov I think this function (TMath::TwoPi()) is constexpr only in ROOT6. I personally wouldn't care of leaving it and finally give the remaining people a push to ROOT6 - my opinion of ROOT5 couldn't be worse, including that we are still forced to comply with it - but I guess some users might not like this.

Reply:
Only the second line 
``` c++
constexpr Float_t twopi = 2.0 * TMath::Pi();
```
was supposed to be highlighted, the line above is OK also in stupid ROOT5.

Reply:
Had to open a new Pull request. Unfortunately I cannot compile ROOT5 locally, so I couldnt test it. (https://github.com/alisw/AliPhysics/pull/18616)